py4DSTEM/process/diffraction/crystal.py

```diff
@@ -684,6 +688,10 @@ def orientation_plan(
         # Calculate wavelenth
         self.wavelength = electron_wavelength_angstrom(self.accel_voltage)
 
+        # store the radial and intensity scaling to use later for generating test patterns
+        self.radial_power = radial_power
```

Comment:
Looks good, but to match the convention of the other variables, anything related to orientation mapping is currently stored in the format:  orientation_variable_name.  Therefore I suggest changing these variables to be called:
self.orientation_radial_power = radial_power
self.orientation_intensity_power = intensity_power

Reply:
✅